webapp/pages/profile/_id/_slug.vue

```diff
@@ -1,7 +1,11 @@
 <template>
   <div>
-    <ds-space />
     <ds-flex v-if="user" :width="{ base: '100%' }" gutter="base">
+      <ds-flex-item width="100%">
+        <base-card class="profile-header-card">
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-06-25T11:54:25Z" title="Thursday, June 25th 2020, 1:54:25 pm +02:00">Jun 25, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -2,7 +2,7 @@
   <div>
     <ds-flex v-if="user" :width="{ base: '100%' }" gutter="base">
       <ds-flex-item width="100%">
-        <base-card class="profile-header">
+        <base-card class="profile-header-card">
```


This change has broken the Cypress test …

I allowed myself to fix this. Hope this is okay for you @DriesCruyskens !?

Reply:
> [<img alt="DriesCruyskens" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/DriesCruyskens) **Authored by [DriesCruyskens](https://github.com/DriesCruyskens)**
_<time datetime="2020-06-25T16:27:25Z" title="Thursday, June 25th 2020, 6:27:25 pm +02:00">Jun 25, 2020</time>_

---


yes off course! my bad